xchange-core/src/main/java/org/knowm/xchange/service/trade/params/TradeHistoryParamOrder.java

```diff
@@ -0,0 +1,7 @@
+package org.knowm.xchange.service.trade.params;
+
+public interface TradeHistoryParamOrder extends TradeHistoryParams {
+	String getId();
```

Comment:
Minor: rename id to orderId, taking into account that trades are queried

Reply:
fixed